dashboards/xdcr-dashboard.json

```diff
@@ -626,6 +626,76 @@
           "_base": "target"
         }
       ]
+    },
+    {
+      "title": "goxdcr mem size and resident",
+      "_base": "panel",
+      "_targets": [
+        {
+          "datasource": "{data-source-name}",
+          "expr": "sysproc_mem_size{proc=\"goxdcr\"}",
+          "legendFormat": "{data-source-name} {{name}} {{proc}}",
+          "_base": "target"
+        },
+        {
+          "datasource": "{data-source-name}",
+          "expr": "sysproc_mem_resident{proc=\"goxdcr\"}",
+          "legendFormat": "{data-source-name} {{name}} {{proc}}",
+          "_base": "target"
+        }
+      ],
+      "fieldConfig": {
+        "defaults": {
+          "custom": {
+            "fillOpacity": 1
+          },
+          "unit": "bytes"
+        }
+      },
+      "options": {
+        "tooltip": {
+          "mode": "multi"
+        }
+      },
+      "type": "timeseries"
```

Comment:
mode of "multi" and type of "timeseries" are the default options and so are not needed if you'd like to remove them.